ocp_resources/cluster_role_binding.py

```diff
@@ -9,3 +10,33 @@ class ClusterRoleBinding(Resource):
     """
 
     api_group = Resource.ApiGroup.RBAC_AUTHORIZATION_K8S_IO
+
+    def __init__(
+        self,
+        name=None,
+        cluster_role=None,
+        subjects=None,
+    ):
+        super().__init__(name=name)
+        self.cluster_role = cluster_role
+        self.subjects = subjects
+
+    def to_dict(self):
+
+        res = super().to_dict()
+        res.setdefault("roleRef", {})
+        if self.cluster_role:
+            """
+            It's the responsibility of the caller to verify the desired configuration they send.
```

Comment:
This comment is not needed; this is applicable to almost anything passed by callers to the api

Reply:
ACK